exercises/ansible_collections/1-create-collections/README.md

```diff
@@ -0,0 +1,612 @@
+# Exercise 1 - Installing and Creating Collections
+
+## Table of Contents
+
+- [Objective](#objective)
+- [Guide](#guide)
+    - [Step 1: Installing collection from the command line](#step-1-installing-collections-from-the-command-line)
+    - [Step 2: Creating collections from the command line](#step-2-creating-collections-from-the-command-line)
+    - [Step 3: Creating custom plugins](#step-3-creating-custom-plugins)
```

Comment:
While technically right, "plugins" is misleading as long as the word "module" is not in there. People who are not deeply familiar with how collections are set up and ansible works already will wonder where we add modules.

So maybe instead of "Creating custom plugins", how about "Adding custom modules and plugins to the collection"?

Reply:
done